src/constants/rpcActions.ts

```diff
@@ -35,6 +35,7 @@ export enum RPCAction {
   ENABLE_OPERATION_HISTORY = "rpc/identity/historyEnable",
   DOWNLOAD_BACKUP = "rpc/backup/download",
   UPLOAD_BACKUP = "rpc/backup/upload",
+  SAVE_MNEMONIC = "rpc/mnemonic/save",
```

Comment:
Just a TODO for future, I think its good to have a convention in naming the rpc end points here. 
I would say sth like `rpc/SERVICE_NAME/SERVICE_FUNCTION`

Reply:
It's good for now, tbd later.